real_time/surface_src/java_src/Alice/src/test/java/com/roboclub/robobuggy/nodes/planners/WaypointFollowerTest.java

```diff
@@ -0,0 +1,15 @@
+package com.roboclub.robobuggy.nodes.planners;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class WaypointFollowerTest {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Each class should declare at least one constructor](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2381518029/issues/source?bid=2938665&fileBranchId=3199438#l7)